bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/hbasesnapshots/EndToEndIT.java

```diff
@@ -98,20 +107,20 @@ public void setup() throws Exception {
     EnvSetup.initialize();
     properties = TestProperties.fromSystem();
 
-    // Configure inputs
-    hbaseSnapshotDir = properties.getCloudDataDir() + "data/";
-    hashDir = properties.getCloudDataDir() + "hashtable/";
-
-    // Configure output
+    // Configure paths
     workDir = properties.getTestWorkdir(UUID.randomUUID());
-    tempDir = workDir + "temp/";
     syncTableOutputDir = workDir + "output/";
+    fixtureDir = workDir + "hbase-snapshot/";
+    hbaseSnapshotDir = fixtureDir + "data/";
+    hashDir = fixtureDir + "hashtable/";
 
     // Cloud Storage config
     GcpOptions gcpOptions = PipelineOptionsFactory.create().as(GcpOptions.class);
     properties.applyTo(gcpOptions);
     gcsUtil = new GcsUtil.GcsUtilFactory().create(gcpOptions);
 
+    uploadFixture(gcsUtil, SNAPSHOT_FIXTURE_NAME, fixtureDir);
```

Comment:
Should we add a teardown method that cleans up the GCS directory after test is finished?

Reply:
Yes, but that can happen in a separate pr. I want to keep this focused.

Reply:
https://github.com/googleapis/java-bigtable-hbase/pull/3609